### PR TITLE
Switch to Yazur V2 as a submodule

### PR DESCRIPTION
**Pull request by Azurethi**

### CODE REVIEW

**Comment by Azurethi** [comment]
Not 100% sure why the check failed, my local copy builds & runs fine. It's something to do with the fact that I've placed the yazur submodule outside of ``src/assets`` (in ``/``), so it's not finding the modules in ``Yazur/es-modules/yolol/`` (referenced as ``../../../Yazur/es-modules/yolol/`` in ``src/app/services/yazur.service.ts``. The submodule could be moved inside of ``src/assets``, however I'm not sure if that'd end up trying to host the whole yazur repo when you only want the es-modules (or would angular only take what it was asked to include)? 

Worst case: if we *need* to move them to ``src/assets`` and angular would try to include everything, we can use a sym link.
Worst worst case: add an action that copies the files over to before builds
Worst worst worst case: Just ditch the submodule & copy the files manually again.

File: src/app/services/yazur.service.ts [hunk 1]
```diff
@@ -41,6 +38,7 @@ export class Chip {
       } );
     }
     this.setParsed( parsed );
+    this.lineChange.subscribe(console.log);
```
Comment (pascalstockert) [inline src/app/services/yazur.service.ts]: Leftover log

File: src/app/services/yazur.service.ts [hunk 2]
```diff
@@ -61,7 +59,14 @@ export class Chip {
   }
 
   interpret(): any {
-    yazurInterpret( this );
+
+    var chipwait=this.localEnv.global[this.localEnv.chipwaitField].value;
```
Comment (pascalstockert) [inline src/app/services/yazur.service.ts]: We dont use var in these lands 👏 😆

**Review by pascalstockert — CHANGES_REQUESTED** [review]
I actually like the idea of using a symlink, makes stuff quite easy! You wanna work on that?

**Pull request closed without merging**